stdlib/public/core/Unmanaged.swift

```diff
@@ -206,10 +206,10 @@ public struct Unmanaged<Instance: AnyObject> {
   public func _withUnsafeGuaranteedRef<Result>(
     _ body: (Instance) throws -> Result
   ) rethrows -> Result {
-    let (guaranteedInstance, token) = Builtin.unsafeGuaranteed(_value)
-    let result = try body(guaranteedInstance)
-    Builtin.unsafeGuaranteedEnd(token)
-    return result
+    var tmp = self
+    let fakeBase: Int? = nil
```

Comment:
I think you should explain what `fakeBase` is for, and why a "real" base isn't actually required in this case.

Reply:
The idea is that the user is stating with this API that they are putting a _fixLifetime around as per the documentation.

Reply:
@atrick I am happy to do that in a follow on commit.